src/Clerk.ts

```diff
@@ -330,21 +330,16 @@ export default class Clerk {
         Logger.debug(`cookieToken: ${cookieToken}`);
         const clientUat = cookies.get('__client_uat');
         Logger.debug(`clientUat: ${clientUat}`);
-        let headerToken = (req.headers['Authorization'] ||
-          req.headers['authorization']) as string;
+        let headerToken = req.headers.authorization;
```

Comment:
:question: Unrelated to this PR, this is vanilla Node.js right? Why are we using the `Request` type from expressjs?

Reply:
@chanioxaris Have we tested that this is case insensitive in popular node frameworks such as Express, Next or Fastify?

Reply:
The `req` object should make it case-insensitive. Express offers the `req.get('authorization')` and `req.header('authorization')` which would theoretically work. I'm guessing the same happens in Node.js? A simple test with 

```
curl -H 'Authorization: Foo'
curl -H 'authorization: Foo'
```
will do.

Reply:
I have test it with a Next.js app for the moment, i will also need to have a look into Express and Fastify.

Reply:
@SokratisVidros Tested with Next.js (`v11.0.1`), Express (`4.17.1`) and Fastify (`v3.25.0`) and working as expected